applications/portal/frontend/src/components/CustomAccordionSummary.jsx

```diff
@@ -88,20 +88,21 @@ const CustomAccordionSummary = ({
                         vertical: 'bottom',
                         horizontal: 'left'
                     }}
+                    disabled={isChild ? true : false}
```

Comment:
This is not very nice in terms of UI. We don't want to have a button marked as disabled without any UI feedback. It's better to omit the button.

It also looks to me that the expression could be simplified to just isChild